doc/joss/calibration-module/paper.md

```diff
@@ -0,0 +1,137 @@
+We present a module for calibrating impact functions based on such data using established calibration techniques like Bayesian optimization.
+It is implemented as submodule of the climate risk modeling platform CLIMADA and fully integrates into its workflow.
+
+# Statement of need
+
+Natural hazards like storms, floods, droughts, and extreme temperatures will be exacerbated by climate change.
+In 2022 alone, natural disasters affected 185 million people and caused economic losses of more than US$ 200 billion [@cred_2022_2023].
+Forecasting the impacts of imminent events as well as determining climate risk according to future socio-economic pathways is crucial for decision-making in societal, humanitarian, political, socio-economic, and ecological issues [@smith_human_2014].
+One major source of uncertainty in such computations is the vulnerability [@rougier_risk_2013].
+Typically modeled as impact function that yields the percentage of affected exposure depending on hazard intensity, vulnerability is difficult to determine *a prioi*.
+Using hazard footprints, exposure, and recorded impacts from past events, researchers therefore employ calibration techniques to estimate unknown impact functions and use these functions for future risk projections or impact forecasts [@eberenz_regional_2021; @luthi_globally_2021; @welker_comparing_2021; @roosli_towards_2021; @kam_impact-based_2023; @Schmid2023a; @riedel_fluvial_2024].
+
+CLIMADA is a widely used framework for calculating weather- and climate-related impacts and risks [@aznar-siguan_climada_2019; @gabriela_aznar_siguan_2023_8383171].
+The aforementioned works calibrated impact functions with different optimization algorithms within the CLIMADA ecosystem, but lack a consistent implementation of these algorithms.
+OASIS, another well-adopted loss modeling framework, also does not feature tools for impact function calibration, and regards vulnerability solely as model input [@oasis].
+With the proposed calibration module, we aim at unifying the approaches to impact function calibration, while providing an extensible structure that can be easily adjusted to particular applications.
+
+# Module Structure
+
+Calibrating the parameters of impact functions can be a complex task involving highly non-linear objective functions, constrained and ambiguous parameters, and parameter spaces with multiple local optima.
+To support different iterative optimization algorithms, we implemented an `Optimizer` base class from which concrete optimizers employing different algorithms are derived.
+All optimizers receive data through a common `Input` object, which provides all necessary information for the calibration task.
+This enables users to quickly switch between optimizers without disrupting their workflow.
+Pre-defined optimizers are based on the `scipy.optimize` module [@virtanen_scipy_2020] and the `BayesianOptimization` package [@nogueira_bayesian_2014].
```

Comment:
We already mention that the module is easily extensible with new optimizers. I would rather leave out your suggestion because it does align worse with the beginning of the next sentence ("The latter is...")